2016-08-02-Bitfinex Article

EvgDubrovin opened the pull request.

JediFaust commented:
/qualitycheck

EvgDubrovin commented:
> `The trading price of Bitcoin declined by 20% after the Bitfinex hack.` ❌

added a source link (https://www.coindesk.com/markets/2016/08/02/bitcoin-drops-nearly-20-as-exchange-hack-amplifies-price-decline/)

EvgDubrovin commented:
> `Bitfinex implemented BitGo's multiple-signature security system to safeguard the funds.` ❌

added a source link (https://en.wikipedia.org/wiki/2016_Bitfinex_hack)

EvgDubrovin commented:
> `The identity of the thief still remains unknown.` ❌

according to the source I attached, it's true (https://www.wired.com/story/security-lapses-at-hacked-crypto-exchange-bitfinex/#:~:text=But%20the%20perpetrator%20of%20the%20hack%20is%20still%20at%20large.%C2%A0). Lichtenstein and his wife are only launderers, they aren't charged in the hack itself.

EvgDubrovin commented:
> `The stolen funds were consolidated into one wallet address: bc1qazcm763858nkj2dj986etajv6wquslv8uxwczt.` ❌

It's written in a plenty of articles. Here is the link to one of them (https://blog.merklescience.com/hacktrack/hack-track-bitfinex-hack-2016-recent-fund-movement-analysis#:~:text=more%20than%2094,address%20is%20bc1qazcm763858nkj2dj986etajv6wquslv8uxwczt.)

EvgDubrovin commented:
> `Bitfinex resumed trading and restored service on August 10, 2016, with additional security measures.` ❌

Bitfinex wrote about it in their announcement, the link to which I attached (https://blog.bitfinex.com/announcements/site-update/#:~:text=to%20restore%20service%20after%20the%20incident%20on%20August%202nd.%20Exchange%20trading%20will%20be%20enabled%20for%20all%20currencies%20and%20pairs%2C%20while%20deposits%20and%20withdrawals%20will%20be%20enabled%20for%20BTC%2C%20ETC%2C%20ETH%2C%20and%20USD%20%E2%80%93%20with%20LTC%20and%20Tether%20to%20follow%20shortly%20thereafter.)

EvgDubrovin commented:
> `Bitfinex announced full recovery from the hack on April 3, 2017, by redeeming all issued and outstanding BFX tokens.` ❌

It's written in the link I attached: (https://www.bitfinex.com/posts/198#:~:text=redeeming%20100%25%20of%20all%20currently%20issued%20and%20outstanding%20BFX%20tokens.%20This%20will%20be%20the%20final%20redemption%20of%20BFX%20tokens) and (https://www.bitfinex.com/posts/198#:~:text=full%20recovery%20in%20just%20eight%20months.)

EvgDubrovin commented:
> `Bitfinex received stolen BTC from the U.S. government on February 25, 2019.` ❌

It's written in the article I attached (https://medium.com/bitfinex/bitcoins-returned-to-bitfinex-by-u-s-government-51fe84e8bb12#:~:text=Bitfinex%20are%20pleased%20to%20announce%20that%2027.66270285%20BTC%20that%20were%20unlawfully%20taken%20in%20the%20August%202016%20security%20breach%20have%20been%20received%20by%20Bitfinex%20from%20the%20U.S.%20government%20as%20the%20result%20of%20U.S.%20law%20enforcement%20efforts.) and in many other sources.

EvgDubrovin commented:
> `Bitfinex offered a $400 million reward on August 4, 2020, for information leading to the identification of hackers involved in the attack.` ❌

It's written in the source I attached: (https://www.bitfinex.com/posts/494#:~:text=Bitfinex%20is%20offering%20a%20reward%20to%20any%20persons%20that%20connect%20us%20with%20hackers%20responsible%20for%20the%20unauthorized%20transfer) and (https://www.bitfinex.com/posts/494#:~:text=The%20aggregate%20rewards%20available%20under%20this%20programme%20could%20be%20worth%20up%20to%20approximately%20US%24400%20million%20at%20the%20current%20BTC%20price). It's the Bitfinex's statement.

EvgDubrovin commented:
> `Bitfinex recovered stolen BTC in collaboration with Poloniex on December 14, 2021.` ❌

It's written in the source I attached (https://www.bitfinex.com/posts/746#:~:text=The%20recovery%20of%20the%20stolen%20BTC%20was%20done%20in%20collaboration%20with%20Poloniex). It's the Bitfinex's statement.

EvgDubrovin commented:
> `94,643.29 BTC, more than 79% of the stolen funds, were transferred and consolidated into a single wallet on February 1, 2022.` ❌

It's written in the source I attached (https://news.bitcoin.com/3-6-billion-in-bitcoin-from-the-2016-bitfinex-hack-consolidates-into-a-single-address/#:~:text=More%20Than%2079%25%20of%20the%202016%20Stolen%20Bitfinex%20Bitcoin%20Moves%20Into%201%20Address). It's the bitcoin.com article with a variety of other sources.

EvgDubrovin commented:
> `The U.S. Department of Justice recovered approximately 94,636 stolen BTC on February 7, 2022.` ❌

It's written in the source I attached - the  U.S. Department of Justice statement of facts. See point 52. Also, there are other sources confirming this info.

JediFaust commented:
<img width="539" alt="image" src="https://github.com/1712n/dn-institute/assets/83595409/cd7bd6b4-b351-404c-827c-80d5532adf7b">

JediFaust commented on content/attacks/posts/2016-08-02-Bitfinex.md:
```diff
@@ -0,0 +1,38 @@
+
+## Security Failure Causes
+
+**Violation of the Security Protocol:** According to OCCRP report, Bitfinex [failed](https://www.occrp.org/en/blog/17670-confidential-report-flags-bitfinex-security-lapses-in-huge-2016-hack) to implement operational, financial, and technological controls proposed by its digital security partner Bitgo. Critical errors were made, such as storing two out of three security keys on the same device, pr ...
```
Remove last sentence

EvgDubrovin replied:
Done

JediFaust reviewed (changes requested):
Please, simplify and shorten the whole text, since there are unnecessary details. Also try to rewrite some parts with your own words, to pass plagiarism checks.

EvgDubrovin commented:
> Please, simplify and shorten the whole text, since there are unnecessary details. Also try to rewrite some parts with your own words, to pass plagiarism checks.

1. Rewrote the whole text in the last commit.
2. Removed uninformative parts (mostly from the Timeline).
3. Updated the Headers in accardance with the latest requirements.

JediFaust commented:
<img width="539" alt="image" src="https://github.com/1712n/dn-institute/assets/83595409/50c1f612-81fd-439e-b6d7-4474fffbc603">
Plagiarism issue still remains

EvgDubrovin commented on content/attacks/posts/2016-08-02-Bitfinex.md:
```diff
@@ -0,0 +1,37 @@
+Date: 2016-08-02
+Type of attack: "Custodian"
+Target entity: "Bitfinex"
+Type of entity: "Exchange"
```
Done

EvgDubrovin commented:
There is no more plagiarism, according to Grammarly.

JediFaust reviewed (changes requested):
Please, use [language tool](https://languagetool.org/) and fix grammar issues. Also, merge main into your branch

EvgDubrovin commented:
> Please, use [language tool](https://languagetool.org/) and fix grammar issues. Also, merge main into your branch

Here is the Language Tool [report](https://drive.google.com/file/d/1180ModmqdiTIQuBnfbKP9AVeUDNDyC4x/view?usp=sharing). 
- I removed the spurious comma. 
- But Language Tool considers "BitGo" and "OCCRP" as spelling mistakes, though these are the names of organizations. 
- I merged main into Bitfinex. There's "Already up to date."

JediFaust commented on content/attacks/posts/2016-08-02-Bitfinex.md:
```diff
@@ -0,0 +1,37 @@
+---
+date: 2016-08-02
+target-entities: Bitfinex
+entity-types: Exchange
```
Add Custodian as an entity type

EvgDubrovin replied:
Done

JediFaust commented on content/attacks/posts/2016-08-02-Bitfinex.md:
```diff
@@ -0,0 +1,37 @@
+date: 2016-08-02
+target-entities: Bitfinex
+entity-types: Exchange
+attack-types: Custodian
```
Attack type for the case is `Web Infrastructure Attack'

EvgDubrovin replied:
Done

JediFaust reviewed (changes requested):
Please, also merge main with the branch

EvgDubrovin commented:
> Please, also merge main with the branch

Done

JediFaust commented on content/attacks/posts/2016-08-02-Bitfinex.md:
```diff
@@ -0,0 +1,40 @@
+target-entities: 
+- Bitfinex
+entity-types: 
+- Custodian
```
`entity-types` should look like 
> - Custodian
> - Exchange

so, return 'Exchange' back please

JediFaust reviewed (changes requested):
Please, merge main into the branch

JediFaust commented:
/payout
